internal/apps/poll/restrict.go

```diff
@@ -121,6 +144,15 @@ func RestrictVote(r restricter.HasPermer, canSee, canManage string) restricter.E
 			return element, nil
 		}
 
+		var delegatedUserID int
```

Comment:
Please use a shorter variable name here. For example `duid`.

Reply:
This matches the check above since `userID` is used there.